packages/blocks/src/editor-blocks-renderer/index.tsx

```diff
@@ -10,14 +12,13 @@ type EditorBlocksRendererProps = {
 /**
  * A react component to render editor blocks.
  * @param props - Props.
- * @param props.blockDefinitions - blocks rendering functions.
  * @param props.editorBlocks - A list of blocks to be rendered.
  * @return The rendered template
  */
 export default function EditorBlocksRenderer( {
 	editorBlocks,
-	blockDefinitions,
 }: EditorBlocksRendererProps ) {
```

Comment:
Pros vs cons about leaving keeping this prop, and then having a hierarchy of `props?.blockDefinitions || getConfig()?.blockDefinitions || defaultBlockDefinitions`

Reply:
Would you please tell me a case where the user would want to overwrite the block definitions passed from the config file by passing them as props to `EditorBlocksRenderer`?
I can't think of any pros of doing this at the moment.
Also, we are doing something similar to `props?.blockDefinitions || defaultBlockDefinitions` here:
https://github.com/rtCamp/snapwp/blob/806792d153651dc2355ce58876fe251e9cb1de7a/packages/blocks/src/block-manager/index.tsx#L47-L70